README.md

```diff
@@ -21,136 +20,157 @@
+jpg,                ,                         0x
+
+video formats
+mp4,                ,                         0x
+mkv,                ,                         0x
+
+blockchain formats
+
+VCS formats
+
+IPLD formats
+dag-pb,             MerkleDAG protobuf,       0x70
+dag-cbor,           MerkleDAG cbor,           0x71
+eth-block,          Ethereum Block (RLP),     0x90
+eth-tx,             Ethereum Tx (RLP),        0x91
+bitcoin-block,      Bitcoin Block,            0xb0
+bitcoin-tx,         Bitcoin Tx,               0xb1
+stellar-block,      Stellar Block,            0xd0
+stellar-tx,         Stellar Tx,               0xd1
 ```
 
-These path names happen to be resolvable -- not just in a "multicodec muxer(e.g [multistream]())" but -- in the internet as a whole (provided the program (or OS) knows how to use the `/ipfs` and `/http` protocols).
-
 ## Implementations
 
-- [go-multicodec](https://github.com/jbenet/go-multicodec)
-- [go-multistream](https://github.com/whyrusleeping/go-multistream) - Implements multistream, which uses multicodec for stream negotiation
-- [js-multistream](https://github.com/multiformats/js-multistream) - Implements multistream, which uses multicodec for stream negotiation
-- [clj-multicodec](https://github.com/greglook/clj-multicodec)
-
+- [go](https://github.com/multiformats/go-multicodec/)
+- [JavaScript](https://github.com/multiformats/js-multicodec)
+- [Add yours today!](https://github.com/multiformats/multicodec/edit/master/multicodec.md)
 
 ## FAQ
 
-> **Q. Why?**
+> **Q. I have questions on multicodec, not listed here.**
 
-Today, people speak many languages, and use common ones to interface. But every "common language" has evolved over time, or even fundamentally switched. Why should we expect programs to be any different?
+That's not a question. But, have you checked the proper [multicodec FAQ](./README.md#faq)? Maybe your question is answered there. This FAQ is only specifically for multicodec-packed.
 
-And the reality is they're not. Programs use a variety of encodings. Today we like JSON. Yesterday, XML was all the rage. XDR solved everything, but it's kinda retro. Protobuf is still too cool for school. capnp ("cap and proto") is
-for cerealization hipsters.
+> **Q. Why?**
 
-The one problem is figuring out what we're speaking. Humans are pretty smart, we pick up all sorts of languages over time. And we can always resort to pointing and grunting (the ascii of humanity).
+Because [multistream](https://github.com/multiformats/multistream) is too long for identifiers. We needed something shorter.
 
-Programs have a harder time. You can't keep piping json into a protobuf decoder and hope they align. So we have to help them out a bit. That's what multicodec is for.
+> **Q. Why varints?**
 
-> **Q. Why "codec" and not "encoder" and "decoder"?**
+So that we have no limitation on protocols. Implementation note: you do not need to implement varints until the standard multicodec table has more than 127 functions.
```

Comment:
yes, we probably need to remove this, and add multicodec impls to all the use-case specific multiformats IF their table requires it.